package.json

```diff
@@ -38,7 +38,7 @@
     "shelljs": "0.8.5",
     "shx": "0.3.4",
     "snyk": "^1.364.0",
-    "ts-loader": "5.4.5",
+    "ts-loader": "7.0.3",
```

Comment:
This was such a pain.  Eventually in the end I found an issue in ts-loader where it was failing to resolve json modules (even though tsc was compiling fine).